ors-config.yml

```diff
@@ -0,0 +1,40 @@
+##### openrouteservice settings file #####
+# This file contains parameters for openrouteservice.
+# For a full list of possible parameters see https://giscience.github.io/openrouteservice/run-instance/configuration/
+
+ors:
+  engine:
+    source_file: /home/ors/files/data.osm.pbf
```

Comment:
suggestion(non-blocking): Would it be possible to set the service user's `$HOME` to `/var/lib/ors/` and configure this to reference `/var/lib/ors/osm/data.osm.pbf`?

Reply:
What would the benefit of that be? I did it the way that I did because that lined up with the ["Running with Docker" docs](https://giscience.github.io/openrouteservice/run-instance/running-with-docker), so I'm a little worried about creating confusion by drifting away from that without knowing why.

Reply:
I think it's more practical and accepted for services to be run out of `/var/lib` instead of `/home` (which is for users). I picked this up from how NixOS configures services, so I can't really speak to why we shouldn't use `/home`.

So _IMO_, this falls under "linux best practices", which _I think_ docker containers ignores a bunch of.

Reply:
Oh, apparently, OSM makes some assumptions >.>
https://hub.docker.com/layers/openrouteservice/openrouteservice/v8.0.0/images/sha256-1de021aa9f1c7bf773ea3beb161d981caee85bc1f5dd68d47e75cef850acf381?context=explore